httpserver/CMakeLists.txt

```diff
@@ -10,7 +10,8 @@ set(HEADER_FILES
     service_node.h
     serialization.h
     rate_limiter.h
-    ../external/json.hpp)
+    ../external/json.hpp
+    https_client.h)
```

Comment:
Should also add `server_certificates.h`?

Reply:
Done, for consistency.